### ISSUE TITLE
Coolant and waste flow monitor valve indicator change

### ISSUE THREAD

**Issue by ArcTrooper210** [MikaylaFischler/cc-mek-scada#345]
Have the valve indicators change from `OPEN` to `CLOSE` when that specific valve is closed. Also maybe change the default, white if valve is open, to green, and when it's closed have it change to yellow or some other color. The image below is what I was kinda thinking (I used photoshop)
![2023-09-28_21 09 04 - Copy](https://github.com/MikaylaFischler/cc-mek-scada/assets/69335775/2a9b60ba-f4d7-4743-9448-918d14f06475)

**Comment by MikaylaFischler** [comment]
I appreciate the suggestion, but I'm going to add a bit of context first. This is going to be a bit verbose of an explanation of design, but here it goes!

A lot of design decisions I took were based around a few self-defined rules, which include these relevant ones below:
1. Keep the coordinator displays static, like actual control room panels. This means things are always in the same place and always visible (no drop-down menus, no tabs, etc).
2. Attempt to follow closely actual guidelines and real control rooms. This strongly influences many things, among them are terminology and indicator color selection.

Number 1 is why the text of indicators lights never change, which is why `OPEN` is what I settled with. There are however the state indicators I have that are used for multi-block states (reactor, boiler, turbine, matrix, SPS, etc) and the unit waste production on the main view screen. Aesthetically I generally prefer indicator lights, but an opened/closed pair of lights would have taken too much space. Putting state indicators here is something I am open to, so I'll draft out an idea when I get the chance (probably after the next update). The reason I am suggesting state indicators is partially because of the next half of this explanation:

Number 2 applies to color selection here. I tried to stick indicators specifically close to this international atomic energy agency document https://inis.iaea.org/collection/NCLCollectionStore/_Public/27/002/27002051.pdf (which is part of many links here https://github.com/MikaylaFischler/cc-mek-scada/wiki/Design-and-Regulatory-References):

> - Red: immediate action required by the operator to correct the fault, with the means of action
being in the control room
> - Yellow: action required, the time delay for action being defined by the nature of the fault and
the location of controls available to the operator
> - White: information indicating a change of state or an automatic action being taken
> - Green: automatic action being taken by the safety systems, Operators must immediately verify
that the action requested by the system was done.

Red and yellow are the ones I've tried most to keep assigned to or associated with the above roles. In a recent update I had changed a few indicators from being red/green to off/green, since the red wasn't actually indicating a problem (units not ready as one example). Green has been my more flexible color, to indicate both connections and safe states. The front panels on computers themselves are a different story (which is going to make #340 a bit more complex), as these and other visual guidelines are only being applied to the main control room view. The front panels on computers are modeled instead after industrial automation hardware.

So, yellow here would imply a problem, rather than a functional state (open vs closed). I had debated for a while what colors and what text I would use here, and this is what came of that. Not ideal, I agree, but it seemed to be the clearest.

In short, white is the only color I can use here (if I want to follow the guidelines I've been following) to indicate a neutral action having been taken (valve state change), unless I use a state indicator rather than an indicator light.

**tl;dr I'll draft up a state indicator in the place of the open indicator and we can see how it ends up looking. It'll come down to the balancing act between being realistic yet also intuitive (alongside every other design choice I've made 😅).**
